views/beneficial-owner-individual.html

```diff
@@ -62,9 +62,9 @@
 
         {{ govukInsetText({
           html: "
-            <h2 class='govuk-heading-m'>Information shown on the public register</h2>
-            <p class='govuk-body'>The beneficial owner's home address will not be shown unless it is the same as their correspondence address. Only the month and year of their date of birth will be shown.</p>
-            <p class='govuk-body'>All other information provided in this section will be shown on the public Register of Overseas Entities.</p>
+            <h2 class='govuk-heading-m'>What information we’ll show on the public register</h2>
```

Comment:
Can we do something like the following https://github.com/companieshouse/overseas-entities-web/blob/566c59c6c4d351eb5f116ebdc9bfd6cf8c013023/views/due-diligence.html#L85 thanks